app.ts

```diff
@@ -138,6 +138,55 @@ app.get('/badge/vscode/:id', c => {
 	return c.body(badge)
 })
 
+app.get('/badge/vscode/:id', c => {
```

Comment:
This is overridden by the code above. See my comment below on your code breaking ultimate and do the same change in the code above. People using Code as their presence name would not work in this instance.